### PR TITLE
Removed stand glitch

### PR DESCRIPTION
**Pull request by CoobMC**
The hologram stand was sometimes visible when creating an animation, so I added a consumer which prevented the stand from glitching.

I also replaced CompMaterial with ItemStack because it allows the stand to hold custom heads.

### CODE REVIEW

File: src/main/java/org/mineacademy/fo/model/SimpleHologramStand.java [hunk 1]
```diff
@@ -33,35 +33,34 @@ public class SimpleHologramStand extends SimpleHologram {
 	 * Create a new simple hologram using armor stand showing the given material
 	 *
 	 * @param spawnLocation
-	 * @param material
+	 * @param item
 	 */
-	public SimpleHologramStand(Location spawnLocation, CompMaterial material) {
```
Comment (kangarko) [inline src/main/java/org/mineacademy/fo/model/SimpleHologramStand.java:38]: Please do not make breaking changes, students watching training videos will get compile issues if the constructor is changes. Can you recode this fix to retain the old CompMaterial constructor, or simply create another one with ItemStack, keeping the old one instead?

**Comment by kangarko** [comment]
Thanks, added a comment before I can approve this.

**Comment by CoobMC** [comment]
Hi Matej, I found a way to keep the original constructor in the SimpleHologramStand class, although I had to add a method in ItemCreator in order to make it work.

I have also modified the SimpleHologram class to allow people to create stands withought having to create a lore (useful for animation creation).

Hope you like these changes. :)

**Comment by kangarko** [comment]
Thanks, pulled manually!

**Pull request closed without merging**